.github/workflows/build.yml

```diff
@@ -532,7 +532,7 @@ jobs:
         uses: actions/setup-dotnet@v3
 
       - name: Run Tests
-        run: dotnet test --filter '${{ matrix.config.filter}}' --configuration release --logger trx --blame --collect:"XPlat Code Coverage" --settings ./.github/workflows/codecov.runsettings --results-directory ./TestResults/
+        run: dotnet test --filter '${{ matrix.config.filter}}' --configuration release
```

Comment:
>release [](http://example.com/codeflow?start=79&length=7)

Don't you need to still specify the settings file? Also, does anything else in the build depend on the results being in `./TestResults`?

Reply:
> Don't you need to still specify the settings file? 
Yes, good catch

> Also, does anything else in the build depend on the results being in ./TestResults?
Yes, the results directory is now configured in the `.runsettings` file